proteuslib/util/flowsheets.py

```diff
@@ -0,0 +1,83 @@
+###############################################################################
+# ProteusLib Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/nawi-hub/proteuslib/"
+#
+###############################################################################
+
+from pyomo.environ import check_optimal_termination, TransformationFactory
+from idaes.core.util import get_solver
+from idaes.core.util.model_statistics import degrees_of_freedom
+from idaes.core.util.scaling import (unscaled_variables_generator,
+                                     unscaled_constraints_generator,
+                                     calculate_scaling_factors)
+from pyomo.util.check_units import assert_units_consistent
+
+''' This utility file was copied over from the flowsheets/util.py 
+used for developing the full treatment train use case'''
+
+# TODO: carried this over to get testing for zero order NF working with minimal modification;
+#  reconsider what to keep
```

Comment:
We shouldn't add this file. These utilities are already mostly included in other util files like the check degrees of freedom is in util/initialization. The solves with user scaling can use the approach of our other unit and property model test.